test/webService/sample_predict_from_df.py

```diff
@@ -0,0 +1,44 @@
+import requests
+
+
+def main():
+    sample_data = {
+      "regressor": "OUTPUT/faas_test/best.pickle",
+      "df": {
+        "Lambda": [
+          0.222486056568079,
+          0.233038487465872,
+          0.487490803901082
+        ],
+        "warm_service_time": [
+          2.00009526463535,
+          1.96695204745981,
+          1.33774671968767
+        ],
+        "cold_service_time": [
+          2.38527585140162,
+          2.30448269192042,
+          1.66541166626772
+        ],
+        "expiration_time": [
+          600,
+          600,
+          600
+        ],
+        "ave_response_time": [
+          2.00151865892106,
+          1.96814934943309,
+          1.33831639990086
+        ]
+      }
```

Comment:
Since this example uses `predict_from_df()`, I would rather start from an actual `pandas.DataFrame`. For example, using this nice one-liner to convert it to a dictionary:
```python
df_dict = {key: vals.tolist() for key, vals in df.iteritems()}
```

Reply:
This is an example of the json you pass as input (not necessarily through python, it may be through postman or other apps), thus I would keep it as is: the idea was that you can copy/paste it on whatever application you use for sending requests